styles.css

```diff
@@ -0,0 +1,96 @@
+
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: auto;
+  font-family: 'Poppins', sans-serif;
+}
+
+nav {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Nice work with the flex-box arrangement as it has been the design requirement, but what is your view on having a specific class named after a flex-box property, then applying the property to wherever applicable, this will make your code clean DRY, more readable and maintainable. 😀 

for example.

```
.flex-row {
  display: flex
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```

if `justify-content` or any property is repeated many times too, you can do it as such.